incubator/solr/templates/service-client.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "solr.fullname" . }}
+  labels:
+    app: {{ template "solr.name" . }}
+    chart: {{ template "solr.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: {{ .Values.ports.client.containerPort }}
+      protocol: TCP
+      name: solr-http
+  selector:
+    app: {{ template "solr.name" . }}
+    release: {{ .Release.Name }}
```

if we add this to the service then there will be no need for ingress since we only need a local IP which we can ssh forward it if needed
```suggestion
    release: {{ .Release.Name }}
 status:
  loadBalancer: {}
```

---

@gadelkareem - Could you give me some more context around your suggestion?  I'm not sure if I understand the change your requesting.

---

You can ignore that comment. I just noticed that a ClusterIP service with `status: {loadbalancer:{}}` is exposed locally to the local server network without the need for a NodePort or ingress but I could not find anything related on this in the docs.